base/js/admin.js

```diff
@@ -808,65 +745,138 @@
     };
 
     // Widgets Bundle utility functions
-    var sowForms = {
-        /**
-         * Get the unique index of a repeater item.
-         *
-         * @param $el
-         * @return {*}
-         */
-        getRepeaterId: function( $el ) {
-            if( typeof this.id === 'undefined' ) {
-                this.id = 1;
-            }
-
-            var $r = $el.closest('.siteorigin-widget-field-repeater-item');
-            if( $r.length ) {
-                var itemId = $r.data('item-id');
-                if( itemId === undefined ) {
-                    itemId = this.id++;
-                }
-                $r.data('item-id', itemId);
-
-                return itemId;
-            }
-            else {
-                return false;
-            }
-        },
-
-        getWidgetFieldVariable: function ( widgetClass, elementName, key ) {
-            var widgetVars = window.sow_field_javascript_variables[widgetClass];
-            // Get rid of any index placeholders
-            elementName = elementName.replace( /\[#.*?#\]/g, '');
-            var variablePath = /[a-zA-Z0-9\-]+(?:\[c?[0-9]+\])?\[(.*)\]/.exec( elementName )[1];
-            var variablePathParts = variablePath.split('][');
-            var elementVars = variablePathParts.length ? widgetVars : null;
-            while(variablePathParts.length) {
-                elementVars = elementVars[variablePathParts.shift()];
-            }
-            return elementVars[key];
-        },
-
-        fetchWidgetVariable: function (key, widget, callback) {
-            window.sowVars = window.sowVars || {};
-
-            if (typeof window.sowVars[widget] === 'undefined') {
-                $.post(
-                    soWidgets.ajaxurl,
-                    { 'action': 'sow_get_javascript_variables', 'widget': widget, 'key': key },
-                    function (result) {
-                        window.sowVars[widget] = result;
-                        callback(window.sowVars[widget][key]);
-                    }
-                );
-            }
-            else {
-                callback(window.sowVars[widget][key]);
-            }
-        }
-    };
-    window.sowForms = sowForms;
+	/**
+	 * Get the unique index of a repeater item.
+	 *
+	 * @param $el
+	 * @return {*}
+	 */
+	sowbForms.getRepeaterId = function( $el ) {
+		if( typeof this.id === 'undefined' ) {
+			this.id = 1;
+		}
+
+		var $r = $el.closest('.siteorigin-widget-field-repeater-item');
+		if( $r.length ) {
+			var itemId = $r.data('item-id');
+			if( itemId === undefined ) {
+				itemId = this.id++;
+			}
+			$r.data('item-id', itemId);
+
+			return itemId;
+		}
+		else {
+			return false;
+		}
+	};
+
+	sowbForms.getWidgetFieldVariable = function ( widgetClass, elementName, key ) {
```

Comment:
Do you want to add comments for all these functions? I can't actually remember what `getWidgetFieldVariable` and some of these other functions do.